### PR TITLE
dev, v3.0, v2.1: add safe-mode config in binlog reparo doc

### PR DESCRIPTION
**Pull request by juliezhang1112**
### What is changed, added or deleted? 
- Add a `safe-mode` config in binlog reparo doc

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/1558

### Which version does your change affect? 
dev, v3.0, v2.1

### Checklist 

- [ ] Add a new file to `TOC.md`
- [ ] No Tab spaces anywhere 
- [ ] Leave a blank line both before and after a code block
- [ ] Keep the first level heading consistent with `title` in metadata
- [ ] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: dev/reference/tools/tidb-binlog/reparo.md [hunk 1]
```diff
@@ -81,6 +83,11 @@ log-level = "info"
 # If it is set to "mysql", you need to configure `host`, `port`, `user` and `password` in [dest-db].
 dest-type = "mysql"
 
+# Safe-mode configuration
+# Value: "true/false" ("false" by default)
+# If it is set to "true", Reparo splits the update statement into a delete plus replace statement.
```
Comment (TomShawn) [inline dev/reference/tools/tidb-binlog/reparo.md]:
```suggestion
# If it is set to "true", Reparo splits the `UPDATE` statement into a `DELETE` statement plus a `REPLACE` statement.
```

File: dev/reference/tools/tidb-binlog/reparo.md [hunk 2]
```diff
@@ -81,6 +83,11 @@ log-level = "info"
...
+safe-mode = false
```
Comment (TomShawn) [inline dev/reference/tools/tidb-binlog/reparo.md]:
```suggestion
safe-mode = "false"
```

File: dev/how-to/maintain/backup-and-restore.md [hunk 3]
```diff
@@ -53,15 +53,15 @@ To quickly backup and restore data (especially large amounts of data), refer to
 
 - The total size of the exported files is 214G. A single table has 8 columns and 2 billion rows.
 - The cluster topology:
-  - 12 TiKV instances: 4 nodes, 3 TiKV instances per node
-  - 4 TiDB instances
-  - 3 PD instances
+    - 12 TiKV instances: 4 nodes, 3 TiKV instances per node
```
Comment (TomShawn) [inline dev/how-to/maintain/backup-and-restore.md]: To make one list nested in another, indent two spaces from the first letter of the list to be nested. Therefore, the format is here correct and needs no more spaces. FYR: https://guides.github.com/features/mastering-markdown/#syntax

**Comment by lilin90** [comment]
@ericsyh PTAL

File: dev/reference/tools/tidb-binlog/reparo.md [hunk 4]
```diff
@@ -81,6 +83,11 @@ log-level = "info"
 # If it is set to "mysql", you need to configure `host`, `port`, `user` and `password` in [dest-db].
 dest-type = "mysql"
 
+# Safe-mode configuration
+# Value: "true"/"false" ("false" by default)
+# If it is set to "true", Reparo splits the `UPDATE` statement into a `DELETE` statement plus a `REPLACE` statement.
+safe-mode = "false"
```
Comment (lilin90) [inline dev/reference/tools/tidb-binlog/reparo.md]: @TomShawn @juliezhang1112 IMO, `safe-mode = false` is exactly what's in the configuration file. So we don't need `""` here. @ericsyh Is that right?

```suggestion
safe-mode = false
```

File: v2.1/reference/tools/tidb-binlog/reparo.md [hunk 5]
```diff
@@ -81,6 +83,11 @@ log-level = "info"
 # If it is set to "mysql", you need to configure `host`, `port`, `user` and `password` in [dest-db].
 dest-type = "mysql"
 
+# Safe-mode configuration
+# Value: "true"/"false" ("false" by default)
+# If it is set to "true", Reparo splits the `UPDATE` statement into a `DELETE` statement plus a `REPLACE` statement.
+safe-mode = "false"
```
Comment (lilin90) [inline v2.1/reference/tools/tidb-binlog/reparo.md]: ditto

File: v3.0/reference/tools/tidb-binlog/reparo.md [hunk 6]
```diff
@@ -82,6 +84,11 @@ log-level = "info"
 # If it is set to "mysql", you need to configure `host`, `port`, `user` and `password` in [dest-db].
 dest-type = "mysql"
 
+# Safe-mode configuration
+# Value: "true"/"false" ("false" by default)
+# If it is set to "true", Reparo splits the `UPDATE` statement into a `DELETE` statement plus a `REPLACE` statement.
+safe-mode = "false"
```
Comment (lilin90) [inline v3.0/reference/tools/tidb-binlog/reparo.md]: ditto

**Comment by ericsyh** [comment]
LGTM

**Comment by lilin90** [comment]
@juliezhang1112 Please sign the CLA.

**Comment by juliezhang1112** [comment]
I create a PR just the same as this one due to the CLA error. 
https://github.com/pingcap/docs/pull/1441

**Pull request closed without merging**